.env

```diff
@@ -75,7 +75,7 @@ _APP_MAINTENANCE_INTERVAL=86400
 _APP_MAINTENANCE_RETENTION_EXECUTION=1209600
 _APP_MAINTENANCE_RETENTION_ABUSE=86400
 _APP_MAINTENANCE_RETENTION_AUDIT=1209600
-_APP_USAGE_AGGREGATION_INTERVAL=30
+_APP_USAGE_AGGREGATION_INTERVAL=2
```

Comment:
it's 2 minutes right?

Reply:
Nope 2 seconds, so that database stats happen every 60 seconds